src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,16 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int BALL_AMOUNT = 3;
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball[] balls = new Ball[BALL_AMOUNT];
+        for (int ball = 0; ball < BALL_AMOUNT; ball++) {
+            balls[ball] = lottery.getRandomBall();
+        }
+        for (Ball ball : balls) {
+            System.out.println(ball.toString());
```

Comment:
Can we use just `System.out.println(ball);` here?

Reply:
Sure. Ty a lot for reminding. Just forget for a while that println() under the hood invoke obj.toString().